pkg/cmd/pipeline/translate.go

```diff
@@ -382,6 +382,13 @@ func AddDevAnnotations(ctx context.Context, manifest *model.Manifest, c kubernet
 			oktetoLog.Infof("could not add %s dev annotations due to: %s", devName, err.Error())
 			continue
 		}
+		if repo != "" && app.ObjectMeta().Annotations[model.OktetoRepositoryAnnotation] == repo {
```

Comment:
Should we check into account if the repo is sanitized? In line 393 we are setting the value of the annotation with the sanitized repo 

`app.ObjectMeta().Annotations[model.OktetoRepositoryAnnotation] = removeSensitiveDataFromGitURL(repo)`

so I assume we should check also that value here instead of just `repo`, right?